<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>451</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GrayButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>GrayScale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProsesButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Proses</string>
    </property>
   </widget>
   <widget class="QLabel" name="img2Label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>451</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1266</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProses">
    <property name="title">
     <string>Proses</string>
    </property>
    <addaction name="actionGrayscale"/>
    <addaction name="actionBrightness"/>
    <addaction name="actionSobel"/>
   </widget>
   <addaction name="menuProses"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
